automatic/dotnet-uninstaller/dotnet-uninstaller.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>dotnet-uninstaller</id>
```

Comment:
I think it's easier to leave the ID as it is right now, since it is less complicated to just write "choco install dotnet-uninstaller". I also see many packages on chocolatey.org that have very short IDs.

Reply:
OK for me